breeze

```diff
@@ -1383,27 +1442,77 @@ ${FORMATTED_DEFAULT_PROD_EXTRAS}
 -L, --use-local-cache
         Uses local cache to build images. No pulled images will be used, but results of local
         builds in the Docker cache are used instead.
+"
+}
 
-$(print_star_line)
- Flags for pushing the docker images
-$(print_star_line)
-
+function flag_push_docker_images() {
+       echo "
 -u, --push-images
         After building - uploads the images to DockerHub
         It is useful in case you use your own DockerHub user to store images and you want
         to build them locally. Note that you need to use 'docker login' before you upload images.
 
-$(print_star_line)
- User and repo used to login to github registry
-$(print_star_line)
-
 -D, --dockerhub-user
         DockerHub user used to pull, push and build images. Default: ${_BREEZE_DEFAULT_DOCKERHUB_USER:=}.
 
 -H, --dockerhub-repo
         DockerHub repository used to pull, push, build images. Default: ${_BREEZE_DEFAULT_DOCKERHUB_REPO:=}.
+"
+}
+
+function flags() {
+      echo "
+$(print_line)
+
+Summary of all flags supported by Breeze:
+
+$(print_star_line)
+ Choose Airflow variant
+$(flag_airflow_variants)
+
+$(print_star_line)
+ Choose backend to run for airflow
```

Comment:
Should we unify the key word? like change all `airflow -> Airflow` and `kind -> KinD`?

Reply:
Good point!